resources/lang/de.js

```diff
@@ -30,6 +30,7 @@ class CactbotLanguageDe extends CactbotLanguage {
       FoeRequiem: 'Requiem Der Feinde', // up 0x8b, down 0x8c
       LeadenFist: 'Verbesserter Schlag auf Schlag',
       TechnicalFinish: 'Komplexes Finale',
+      StandardFinish: 'FIXME',
```

Comment:
Can you add these translations? https://xivapi.com/Action/16003?pretty=true&columns=Name,Name_de,Name_en,Name_fr,Name_ja